Apps/Src/Tasks.c

```diff
@@ -60,6 +61,65 @@ OS_SEM Fault_Sem4;
 cell_asic Minions[NUM_MINIONS];
 uint32_t WDog_BitMap = 0;
 
+
+/** Pend_ResourceForCLI
+ * Pends for desired resource that CLI needs
+ * @param resourceNum the resource you want to wait for, look at CLI_Resource enum in CLI.h
+ */
+void Pend_ResourceForCLI(int resourceNum){
```

Comment:
If only the CLI will be calling these functions, should they be in CLI.c instead of Tasks.c?